src/test/java/co/cask/cdc/plugins/sink/CDCBigTableConfigTest.java

```diff
@@ -36,7 +55,7 @@ public void testValidateReference() {
     );
 
     Assertions.assertThatThrownBy(config::validate)
-      .isInstanceOf(InvalidConfigPropertyException.class);
+      .isInstanceOfSatisfying(InvalidConfigPropertyException.class, withProperty(Constants.Reference.REFERENCE_NAME));
```

Comment:
It's simpler to just do:

```
try {
  config.validate()
} catch (InvalidConfigPropertyException e) {
  Assert.assertEquals(Constants.Reference.REFERENCE_NAME, e.getProperty());
}
```

this is just normal java that mirrors how the caller is expected to behave.

Reply:
fixed